daemons/pacemakerd/pcmkd_subdaemons.c

```diff
@@ -495,13 +495,15 @@ start_child(pcmk_child_t * child)
             // Whether we need root group access to talk to cluster layer
             bool need_root_group = TRUE;
 
+#if SUPPORT_COROSYNC
             if (is_corosync_cluster()) {
                 /* Corosync clusters can drop root group access, because we set
                  * uidgid.gid.${gid}=1 via CMAP, which allows these processes to
                  * connect to corosync.
                  */
                 need_root_group = FALSE;
             }
+#endif // SUPPORT_COROSYNC
```

Comment:
This will be interesting for the method idea since pacemakerd currently doesn't need crm_cluster_t. It does link against libcrmcluster though, for stuff like this. BTW this would be slightly more efficient as a global defined at startup rather than checked for every child start.

Reply:
Doing it as a static local variable since it's only used in one place, unless you strongly prefer global